runbot_send_email/models/runbot_build.py

```diff
@@ -29,14 +33,28 @@ class RunbotBuild(models.Model):
     shareissue_link = fields.Char(compute='_shareissue_link')
 
     @api.multi
+    def get_github_links(self):
+        host_gh, owner_gh, repo_gh = False, False, False
+        repo_git_regex = r"((git@|https://)([\w\.@]+)(/|:))" + \
+            r"([~\w,\-,\_]+)/" + r"([\w,\-,\_]+)(.git){0,1}((/){0,1})"
+        match_object = re.search(repo_git_regex, self.repo_id.name)
+        if match_object:
+            host_gh = match_object.group(3)
+            owner_gh = match_object.group(5)
+            repo_gh = match_object.group(6)
+        self.repo_link = "https://" + host_gh + '/' + owner_gh + '/' + repo_gh
+        self.pr_link = self.repo_link + self.branch_id.name.replace(
+            'refs/heads', '/tree').replace('refs', '')
+        self.commit_link = self.repo_link + '/commit/' + self.name[:8]
+
+    @api.multi
     def _host_name(self):
         base_url = self.env['ir.config_parameter'].get_param('web.base.url')
         self.host_name = urlparse(base_url).hostname
 
     @api.multi
     def _repo_name(self):
-        descrip = self.repo_id.name.replace('.git', '').replace(
-            'https://github.com/', '').replace('/', ' / ')
+        descrip = self.repo_link.replace('https://', '').replace('/', ' / ')
```

Comment:
at this time, only replace '/' by ' / ' to add space between owner and repo, this help to better read of the repo name